BMM.Core/GuardedActions/Base/GuardInvoker.cs

```diff
@@ -0,0 +1,96 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using BMM.Core.ExceptionHandlers.Interfaces;
+using BMM.Core.ExceptionHandlers.Interfaces.Base;
+using BMM.Core.GuardedActions.Base.Interfaces;
+
+namespace BMM.Core.GuardedActions.Base
+{
+    public class GuardInvoker : IGuardInvoker
```

Comment:
do I understand it correctly that you didn't include a Nuget package and instead implemented it directly in BMM.Core?

Reply:
talked about it